### PR TITLE
integrate with circleci

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+version: 2
+jobs:
+  build:
+    working_directory: /workdir
+    docker:
+      - image: docker:17.06.0-ce-git
```
Comment (solidnerd) [inline .circleci/config.yml]: we could specify here `docker:stable-git` than we don't need to worry about the docker version.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        environment:
+          IMAGE_NAME: "sameersbn/gitlab"
+
+    steps:
+      - checkout
+
+      - setup_remote_docker:
+          version: 17.06.0-ce
```
Comment (solidnerd) [inline .circleci/config.yml]: Here than `docker:stable`

**Comment by solidnerd** [comment]
Otherwise LGTM !

File: .circleci/config.yml (hunk 1 above)
Comment (sameersbn) [reply .circleci/config.yml]: circle ci explicitly lists the supported docker versions in https://circleci.com/docs/2.0/building-docker-images/#docker-version. I'll update it to `18.05.0-ce` and give it a test. But as you can tell using the `stable` tag could have unintended consequences

**Comment by solidnerd** [comment]
Should we remove travis than ?

**Comment by sameersbn** [comment]
> Should we remove travis than ?
yep, added a commit the removes travis.yml

**Pull request merged**